.github/semantic.yml

```diff
@@ -0,0 +1,7 @@
+# see https://github.com/Ezard/semantic-prs for more information
+
+enabled: true
+commitsOnly: true
+anyCommit: true
```

Comment:
If I understand, this means that only 1 commit will need to be a conventional commit. Don't we want all of them to follow the convention (except for merge and reverts)?

Reply:
I think that I'm right, because if you see PR #42, it was validated even if it has some commit messages from GitHub which don't follow conventional commit (b485ba4)

Reply:
I think you are right.